### DISCUSSION TITLE
Set it as a wallpaper

### DISCUSSION THREAD

**Discussion by aa89227-zz** [GitSquared/edex-ui#820] [Ideas]
Add an option to set it as a computer wallpaper. Maybe use some key combinations to lock the input.

**Comment by StoiaCode** [comment]
how would you use the terminal then? 
And, furthermore, did you look at the CPU usage this thing has? Even on more modern machines its probably not the best idea. If Athetics is all you want from it I could recommend taking a ~10 minute video of it just idling and putting that into Wallpaper Engine. Saves resources.

**Reply by PoorPocketsMcNewHold** [reply]
Maybe without the terminal prompt/file manager instead. But I would agree with all other points making not useable simply, as a computer animated wallpaper.

**Comment by danbulant** [comment]
On linux you can use the `desktop` property when creating new window (gets ignored elsewhere). I think you should be able to emulate wallpaper on other platforms by always moving the window to the background and not showing it in the app list (just in task bar).

I think I saw something like `desktop` for Mac OS too, but that disables keyboard input (rendering the terminal useless unless you have touchscreen).

**Reply by GitSquared** [reply]
Electron has bindings for making "keep below others" windows on all 3 OS, i believe. But as @EstoyMejor pointed out the app really isn't made for background use. Tbh I don't really see the point.
